### PR TITLE
fix: [CDS-85034]: add css property to fit content in project variable page

### PR DESCRIPTION
**Pull request by MondalSayantan**
### Summary

When there is enough space to show the complete name of the variable, it should be displayed and not shortened

Testing: Only css change so manually tested to check if names are displayed properly

#### Screenshots

Before:
image

After:
image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
- MessageMetadata: `trigger messagecheck`
- GitLeaks: `trigger gitLeaks`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

### Rebase Instructions:
To rebase your branch onto the latest changes in the target branch, simply use the following command in a comment: `/rebase`

### CODE REVIEW

File: src/modules/27-platform/variables/components/VariableResourceModalBody/__test__/VariableResourceModalBody.test.tsx [hunk 1]
```diff
@@ -36,13 +36,17 @@ describe('Secret Resource Modal Body test', () => {
...
         <VariableResourceModalBody {...props}></VariableResourceModalBody>
       </TestWrapper>
     )
     await waitFor(() => getByText('variableLabel'))
-    expect(container).toMatchSnapshot()
+    expect(getByText('variableLabel')).toBeInTheDocument()
+    expect(getByText('typeLabel')).toBeInTheDocument()
+    expect(getByText('platform.variables.inputValidation')).toBeInTheDocument()
+    expect(getByText('valueLabel')).toBeInTheDocument()
+    await waitFor(() => getByText('CUSTOM_VARIABLE'))
```
Comment (chetanvekariya-harness) [inline src/modules/27-platform/variables/components/VariableResourceModalBody/__test__/VariableResourceModalBody.test.tsx:49]: Can we assert on type and value of data as well?

File: src/modules/27-platform/variables/pages/variables/views/__tests__/VariableListView.test.tsx [hunk 2]
```diff
@@ -47,7 +55,11 @@ describe('VariableListView', () => {
       )
 
       await waitFor(() => getByText('CUSTOM_VARIABLE'))
-      expect(container).toMatchSnapshot()
+      const rightArrow = queryByAttribute('data-icon', container, 'arrow-right')
```
Comment (chetanvekariya-harness) [inline src/modules/27-platform/variables/pages/variables/views/__tests__/VariableListView.test.tsx:58]: This is not needed in every test if it covers the use case in any of the test.

File: src/modules/27-platform/variables/pages/variables/views/__tests__/VariableListView.test.tsx [hunk 3]
```diff
@@ -65,22 +77,30 @@ describe('VariableListView', () => {
...
   test('render component at account level - with no variable data', async () => {
-    const { container } = render(
+    const { getByText, queryByText } = render(
       <TestWrapper path={routes.toVariables({ ...accountPathProps })} pathParams={{ accountId: 'dummy' }}>
         <VariableListView variables={{ content: undefined }} openCreateUpdateVariableModal={jest.fn()} />
       </TestWrapper>
     )
 
-    expect(container).toMatchSnapshot()
+    expect(getByText('variableLabel')).toBeInTheDocument()
```
Comment (chetanvekariya-harness) [inline src/modules/27-platform/variables/pages/variables/views/__tests__/VariableListView.test.tsx:100]: This and above test case are exactly same. Even title is same. Let's remove one of these.

File: src/modules/27-platform/variables/components/VariableResourceModalBody/__test__/VariableResourceModalBody.test.tsx (hunk 1 above)
Comment (MondalSayantan) [reply src/modules/27-platform/variables/components/VariableResourceModalBody/__test__/VariableResourceModalBody.test.tsx]: Done

**Pull request merged**